src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java

```diff
@@ -178,4 +116,41 @@ public Navigation openRecentWikiActivity() {
 
     return this;
   }
+
+  public boolean isUserProfileLinkVisible() {
+    try {
+      return userProfile.isDisplayed();
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)